packages/ricos-types/web/src/contexts.ts

```diff
@@ -9,4 +10,5 @@ export interface GeneralContext {
   t: TranslationFunction;
   languageDir: TextDirection;
   theme: RicosTheme;
+  portal: RicosPortal;
```

Comment:
shouldn't it be:

```suggestion
  portal?: RicosPortal;
```

(to sort out all the `null as unknown as RicosPortal` ?)

Reply:
It will fix the unknown, but then every context usage I'll need to check that portal is exist.
I've added doc.body as default value so it will always be defined.